### DISCUSSION TITLE
Версия 2.0 beta 4

### DISCUSSION THREAD

**Discussion by lirix360** [lirix360/ReadmangaGrabber#21] [Announcements]
Новые функции версии 2.0:

- Веб-GUI для скачивания манги. Работает в любом современном браузере.

Изменения beta 4:

- Исправлено скачивание с ReadManga.

Сообщать об ошибках и предложениях [можно в обсуждениях](https://github.com/lirix360/ReadmangaGrabber/discussions).

**Бета-версия доступна только для Windows.**

<hr /><em>This discussion was created from the release <a href='https://github.com/lirix360/ReadmangaGrabber/releases/tag/v2.0beta4'>Версия 2.0 beta 4</a>.</em>

**Comment by COUNTRik** [comment]
Спасибо за эту программу) 
Неделю назад примерно readmanga обновили домен с live на io, программа перестала видеть главы, при этом с mintmanga все видит, так как там домен пока еще не поменяли.

**Reply by lirix360** [reply]
Исправлено в beta 5.
